### PR TITLE
fix: https://perzoinc.atlassian.net/browse/SDA-1610

### PR DESCRIPTION
**Pull request by johankwarnmarksymphony**
https://perzoinc.atlassian.net/browse/SDA-1610

### CODE REVIEW

File: src/app/window-handler.ts [hunk 1]
```diff
@@ -431,6 +431,49 @@ export class WindowHandler {
...
+            let positionY = Math.trunc(display.workArea.y + display.workArea.height / 2 - windowToMove.getBounds().height / 2);
+            if (positionY < display.workArea.y) {
+                positionY = display.workArea.y;
+            }
+
+            logger.info('window-handler: moveWindow, positionX: ' + positionX);
+            logger.info('window-handler: moveWindow, positionY: ' + positionY);
+
+            windowToMove.setPosition(positionX, positionY);
+            // Because of a bug for windows10 we need to call setPosition twice
```
Comment (KiranNiranjan) [inline src/app/window-handler.ts]: @johankwarnmarksymphony Can you please add the link to the issue so, it will be easier to track

**Comment by KiranNiranjan** [comment]
Also, @johankwarnmarksymphony The commit message is missing the ticket number

File: src/app/window-handler.ts [hunk 2]
```diff
@@ -431,6 +431,49 @@ export class WindowHandler {
...
+        const allWindows = BrowserWindow.getAllWindows();
+
+        const parentWindow = allWindows.find((window) => {
+            return (window as ICustomBrowserWindow).winName === 'main';
+        });
+
+        if (parentWindow && windowExists(parentWindow)) {
+            const display = electron.screen.getDisplayNearestPoint({x: parentWindow.getBounds().x, y: parentWindow.getBounds().y});
+
+            logger.info('window-handler: moveWindow, display: ' + JSON.stringify(display.workArea));
```
Comment (keerthi16) [inline src/app/window-handler.ts]:
```suggestion
            logger.info('window-handler: moveWindow', display.workArea);
```

File: src/app/window-handler.ts [hunk 3]
```diff
@@ -431,6 +431,49 @@ export class WindowHandler {
...
+            logger.info('window-handler: moveWindow, windowToMove: ' + JSON.stringify(windowToMove.getBounds()));
```
Comment (keerthi16) [inline src/app/window-handler.ts]:
```suggestion
            logger.info('window-handler: moveWindow, windowToMove:', windowToMove.getBounds());
```

**Pull request merged**